docs/csharp/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md

```diff
@@ -1,379 +1,371 @@
+    - `SumPageSizes`, which gets a list of webpage URLs from the `_urlList` field and then calls `ProcessUrl` and `DisplayResults` to process each URL.
+    - `ProcessUrl`, which downloads the contents of each URL and returns the contents as a byte array.
+    - `DisplayResults`, which displays the number of bytes in the byte array for each URL.
 
-    Copy the following four methods, and then paste them under the `startButton_Click` event handler in MainWindow.xaml.cs:
+    Copy the following code, and paste it above the `OnStartButtonClick` event handler in *MainWindow.xaml.cs*:
 
     ```csharp
-    private void SumPageSizes()
+    readonly IEnumerable<string> _urlList = new string[]
     {
-        // Make a list of web addresses.
-        List<string> urlList = SetUpURLList();
-
-        var total = 0;
-        foreach (var url in urlList)
-        {
-            // GetURLContents returns the contents of url as a byte array.
-            byte[] urlContents = GetURLContents(url);
+        "https://docs.microsoft.com",
+        "https://docs.microsoft.com/azure",
+        "https://docs.microsoft.com/powershell",
+        "https://docs.microsoft.com/dotnet",
+        "https://docs.microsoft.com/aspnet/core",
+        "https://docs.microsoft.com/windows"
+    };
+    ```
 
-            DisplayResults(url, urlContents);
+    Copy the following methods, and paste them below the `OnStartButtonClick` event handler in *MainWindow.xaml.cs*:
 
-            // Update the total.
-            total += urlContents.Length;
-        }
+    ```csharp
+    void SumPageSizes()
+    {
+        var stopwatch = new Stopwatch();
+        stopwatch.Start();
 
-        // Display the total count for all of the web addresses.
-        resultsTextBox.Text += $"\r\n\r\nTotal bytes returned:  {total}\r\n";
-    }
+        int total = _urlList.Select(url => ProcessUrl(url)).Sum();
 
-    private List<string> SetUpURLList()
-    {
-        var urls = new List<string>
-        {
-            "https://msdn.microsoft.com/library/windows/apps/br211380.aspx",
-            "https://msdn.microsoft.com",
-            "https://msdn.microsoft.com/library/hh290136.aspx",
-            "https://msdn.microsoft.com/library/ee256749.aspx",
-            "https://msdn.microsoft.com/library/hh290138.aspx",
-            "https://msdn.microsoft.com/library/hh290140.aspx",
-            "https://msdn.microsoft.com/library/dd470362.aspx",
-            "https://msdn.microsoft.com/library/aa578028.aspx",
-            "https://msdn.microsoft.com/library/ms404677.aspx",
-            "https://msdn.microsoft.com/library/ff730837.aspx"
-        };
-        return urls;
+        stopwatch.Stop();
+        _resultsTextBox.Text += $"\nTotal bytes returned:  {total:#,#}";
+        _resultsTextBox.Text += $"\nElapsed time:          {stopwatch.Elapsed}\n";
     }
 
-    private byte[] GetURLContents(string url)
+    int ProcessUrl(string url)
```

Comment:
Access modifier?

Reply:
I prefer to omit the default access modifiers.

Reply:
The dotnet/runtime [coding style document](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md) mentions:

> We always specify the visibility, even if it's the default (e.g. private string _foo not string _foo). Visibility should be the first modifier (e.g. public abstract not abstract public).

There is also a roslyn analyzer (IDE0040) for that.

If the analyzer is enabled by default, it may be better to include the modifier. If it's not enabled by default, I'll be okay with either adding it or omitting it.